packages/utils/src/lib/progress.integration.test.ts

```diff
@@ -27,15 +27,15 @@ describe('getSingletonMultiProgressBars', () => {
 
 const taskAName = 'a';
 const bars = getSingletonProgressBars({ progressWidth: 1 });
-const tasks = bars?.['tasks'];
-const progressBuffer = bars?.['logger'].progressBuffer;
+const tasks = bars['tasks'];
+const progressBuffer = bars['logger'].progressBuffer;
 
 describe('getProgressBar', () => {
-  it('should init task', async () => {
+  it('should init task', () => {
     expect(bars.getIndex(taskAName)).toBeUndefined();
     getProgressBar(taskAName);
     expect(bars.getIndex(taskAName)).toBe(0);
-    expect(progressBuffer[0].trim()).toBe(
+    expect((progressBuffer[0] as string).trim()).toBe(
```

Comment:
It's shorter to use `!` (allowed in tests):

```suggestion
    expect(progressBuffer[0]!.trim()).toBe(
```

Reply:
Fixed!